V/VMEC/build_tarballs.jl

```diff
@@ -0,0 +1,45 @@
+using BinaryBuilder
+
+name = "VMEC"
+version = v"1.0.0"
+
+sources = [
+    ArchiveSource("https://gitlab.com/wistell/VMEC2000/-/archive/v1.0.0.tar", "179761d5f02cb530fbf6da4121c394a1d585c49c24391ebd87fe5c014e0addcf"),
+]
+
+script = raw"""
+mkdir -p ${libdir}
+cd ${WORKSPACE}/srcdir/VMEC2000*
+make USE_MKL=1
+install -m755 lib/libvmec_mkl.so ${libdir}
+make USE_MKL=1 vmec_clean ; make USE_MKL=0
+install -m755 lib/libvmec_openblas.so ${libdir}
+make USE_MKL=0 vmec_clean
+"""
+
+platforms = [
+    Platform("x86_64", "linux"; libc = "glibc", libgfortran_version="4.0.0"),
+    Platform("x86_64", "linux"; libc = "glibc", libgfortran_version="5.0.0"),
+]
+#platforms = expand_gfortran_versions(platforms)
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libvmec_mkl", :libvmec_mkl, dont_dlopen = true),
+    LibraryProduct("libvmec_openblas", :libvmec_openblas, dont_dlopen = true),
```

Comment:
Why don't dlopen?

Reply:
This was to prevent attempts to open the `mkl` linked library if the appropriates `mkl` libraries are not loaded on the host system.  The `__init__` function of the accompanying `VMEC.jl` package checks to see if the `mkl` is available, as that tends to lead to slight to significant performance gains.  If there is a better way of doing this, I am happy to modify.

Reply:
Uhm, I see, you directly check in your package what to dlopen.  That's ok.

Well, a better approach is arguably to link to [`libblastrampoline`](https://github.com/JuliaLinearAlgebra/libblastrampoline), and use the mechanism in Julia v1.7 to easily switch BLAS backend: https://julialang.org/blog/2021/11/julia-1.7-highlights/#libblastrampoline_mkljl.  Uhm, however they probably also use LAPACK, I don't think that's covered by libblastrampoline.  Alright.